app/src/views/private/components/export-sidebar-detail/export-sidebar-detail.vue

```diff
@@ -65,13 +66,16 @@ export default defineComponent({
 			required: true,
 		},
 	},
+
 	setup(props) {
 		const { t } = useI18n();
 
 		const format = ref('csv');
 		const useFilters = ref(true);
+		const collectionsStore = useCollectionsStore();
+		let collectionName = collectionsStore.getCollection(props.collection).name;
 
-		return { t, format, useFilters, exportData };
+		return { t, format, useFilters, exportData, collectionName };
```

Comment:
One last thing: We should make this a `computed` value so it updates once the `collection` prop changes, otherwise it only fetches it once on setup, and doesn't update when the collection prop changes 🙂

Reply:
```
const collectionName = computed(() => collectionsStore.getCollection(props.collection).name);
```